lib/apachetest.pm

```diff
@@ -42,7 +42,9 @@ sub setup_apache2 {
     my %args = @_;
     my $mode = uc $args{mode} || "";
     # package hostname is available on sle15+ and openSUSE, on <15 it's net-tools
-    my @packages = qw(apache2 /bin/hostname);
+    my @packages = qw(apache2 hostname);
+
+    push @packages, 'apache2-utils' if is_tumbleweed;
```

Comment:
What for?

Reply:
It is not pulled by default http://kepler.suse.cz/tests/6183#step/openssl_tlsv1_3/14

```
   ~ rpm -qf /usr/bin/gensslcert
apache2-utils-2.4.48-1.1.x86_64
```

Reply:
```suggestion
    # For gensslcert
    push @packages, 'apache2-utils' if is_tumbleweed;
```